containers/Loan/List/index.tsx

```diff
@@ -18,30 +17,30 @@ interface Props {
   loadAnalyticsData?: (tinlake: any) => Promise<void>;
   loadUserProxies?: () => Promise<void>;
   auth?: AuthState;
-  analytics?: AnalyticsState
+  analytics?: AnalyticsState;
 }
 
 class LoanList extends React.Component<Props> {
-  componentWillMount() {
-    const { loadLoans, loadAnalyticsData, loadUserProxies, tinlake } = this.props
+  componentDidMount() {
+    const { loadLoans, loadAnalyticsData, loadUserProxies, tinlake } = this.props;
     loadLoans && loadLoans(tinlake);
     loadAnalyticsData && loadAnalyticsData(tinlake);
     loadUserProxies && loadUserProxies();
   }
 
   render() {
     const { loans, analytics, auth, tinlake: { ethConfig: { from: ethFrom } } } = this.props;
-    const user = auth && auth.user
+    const user = auth && auth.user;
     const proxies =  user && user.proxies || [];
-    const availableFunds = analytics && analytics.data && analytics.data.availableFunds || 0;
+    const availableFunds = analytics && analytics.data && analytics.availableFunds || '0';
```

Comment:
same availableFunds has to be part of data

Reply:
Changed